streamdeck_ui/model.py

```diff
@@ -32,6 +34,11 @@ class ButtonState:
     """Font size of the text on the button"""
     background_color: str = ""
     """Background color of the button"""
+    plugin: Plugin = None
```

Comment:
if this will be never saved then we probably do not want it here. instead should be place in the api

Reply:
Gonna move it into the api then, didnt know where to really put it.
When putting it into the api how should I create the reference between the button state and the saved plugin?
Just create a Dict with the serial number, button id, page id and the plugin?

Reply:
Yeah API will need store the references of the plugins, perhaps instead  the data/config models you can create some model for the API that really represents the actual state in the API of each button. in that place you can do more complex stuff like this storing in memory properties

that also simpifies future implementations like allow change the button state without writing that to the config